practice-app/server/src/app.js

```diff
@@ -12,7 +12,7 @@ app.use(
 
 app.use(express.json());
 
-morganBody(app);
+// morganBody(app);
```

Comment:
I could not be sure whether you commented this on purpose or for testing purposes.

Reply:
I commented this for testing. I now made it parametric - it can be switched off from .env (LOG)